mithril-common/Cargo.toml

```diff
@@ -54,6 +54,9 @@ mithril-stm = { path = "../mithril-stm", default-features = false, features = ["
 slog-scope = "4.4.0"
 
 [features]
+default = ["allow_uncertified_signer_registration"]
 portable = ["mithril-stm/portable"]
 test_only = []
 allow_skip_signer_certification = []
+allow_uncertified_signer_registration = []
```

Comment:
We could use the existing flag we have for this, right? The `allow_skip_signer_certification`, or would it be too complex to achieve?

Reply:
That's a good idea, it will make things easier to read and maintain :+1: 
I will add a commit that merges the two features.